content/blog/get-started-with-application-performance-monitoring-tools-overview.md

```diff
@@ -0,0 +1,235 @@
+
+Prometheus is an open source tool with *46K* GitHub stars and *7.7K* Github forks. In [StackShare community](https://stackshare.io/prometheus), Prometheus has been mentioned in *852* company stacks and *1962* developer stacks. Since it is free, Prometheus certainly wins on pricing. However, full functionality of Prometheus demands skills in open source and competence in _Apache_ based applications. Without those required skills and experience, the Prometheus interface can be difficult to master, and some others even find it difficult to set it up and scale.
+
+Prometheus is maintained by volunteers, not by a company. It relies on other open source tools for security. Fixing security issues in Prometheus is done on a *best-effort* basis. Prometheus strives to release security fixes within 7 days for its key components *alertmanager*, *node exporter*, *blackbox exporter* and *pushgateway*, etc.
+
+### **Apache SkyWalking**
+
+![](/img/skywalking.png)
+
+[Apache SkyWalking](https://skywalking.apache.org/) is an open source APM tool with capabilities for monitoring, tracing and diagnosing distributed system. It’s especially designed for microservices, cloud native and container-based architectures. 
+
+Key features:
+
+* provide metrics analysis of services, service instances and endpoints with distributed tracing, log collecting and metrics collecting and customization;
+* support root cause analysis with profiling the code on the runtime by in-process agent, _eBPF_ profiler and network profiler;
+* provide dependency analysis of service instances and endpoins; 
+* support service topology map analysis;
+* detect slow services and endpoints and provide performance optimization;
+* detect slow _SQL_ statement for database performance monitoring;
+* provide message queue performance and consuming latency monitoring;
+* start tracing from browser for browser performance monitoring;
+* support infrastructure monitoring for Kubernetes and Linux;
+* support alerting using rules in both observability analysis language and metric analysis language;
+
+Apache SkyWalking provides a list of agents to be used for building *Java*, *.NET Core*, *PHP*, *Node.js*, *Golang*, *LUA*, *Rust* and *C++* apps. It supports to integrate and collect data from multiple sources, including _Prometheus_, _OpenTelemetry_ and _Zabbix_ for metrics and logs, _Zipkin_ for traces. It provides tracing, metrics analysis, alerting, service mesh observability and visualization. 
+
+Apache SkyWalking is an open source tool with *21K* GitHub stars and *6K* GitHub forks. In [StackShare community](https://stackshare.io/apache-skywalking#stacks.), Apache SkyWalking does not yet have many approval, only being  mentioned in 12 developer stacks. However, Apache SkyWalking has more than *600* contributors on GitHub and thousands of contributions every year. All the agents for application instrumentation have been actively maintained. 
```

Comment:
In the first sentence, change "does not yet have many approval" to something like "does not yet have much approval" or "does not yet have as broad an acceptence, only being..." or "does not yet have as great a share of the market..." (that last one might be the most appropriate.